### PR TITLE
Qaart 693

### PR DESCRIPTION
**Pull request by ludwikkazmierczak**
Added a possibility to mock ads using cookie "mock-ads" by setting a param:
```
-DmockAds=true
```

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 19 issues:
* MAJOR 10 major
* MINOR 9 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Define a constant instead of duplicating this literal "<tr class=\"" 5 times.](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L78) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L174) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation.](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L384) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)
* MAJOR [Remove those useless parentheses.](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L183) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessParenthesesCheck)
* MAJOR [Replace this usage of System.out or System.err by a logger.](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L426) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)
* MAJOR [Remove this unused "date" local variable.](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L392) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)
* MINOR [Define a constant instead of duplicating this literal ".html'>HTML Source</a></td></tr>" 3 times.](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L81) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal ".html" 3 times.](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L75) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal ".png'>Screenshot</a><br/><a href='screenshots/screenshot" 3 times.](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L80) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Introduce a new variable instead of reusing the parameter "imageAsBase64".](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L181) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1226)
* MINOR [Define a constant instead of duplicating this literal "\"><td>" 5 times.](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L78) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Remove this unused "dateFormat" local variable.](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L391) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L425) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L343) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MINOR [Define a constant instead of duplicating this literal "</td><td> <br/> &nbsp;</td></tr>" 6 times.](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L143) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "error" 4 times.](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L76) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Replace this usage of System.out or System.err by a logger.](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L323) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)
* MINOR [Define a constant instead of duplicating this literal "success" 4 times.](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L76) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "</td><td>" 5 times.](https://github.com/Wikia/selenium-tests/blob/f5725fa8e4f681eb6fd94dd4011c55b753ed9bed/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L78) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)

**Comment by tomasznapieralski** [comment]
Awesome!

File: src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java [hunk 1]
```diff
@@ -187,4 +196,6 @@ public static Dimension getBrowserSize() {
 
     return res.toArray(new String[res.size()]);
   }
+
+
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java]: remove those 2 lines

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**